tools/nl/nodejs_query_differ/differ.py

```diff
@@ -21,6 +22,12 @@
 
 _GCS_BUCKET = 'datcom-website-periodic-testing'
 _OUTPUT_FILE = 'differ_results.json'
+_GOLDEN_FOLDER = 'golden'
+_DIFF_SUCCESS_MSG = 'Success'
+_EMAIL_SUBJECT_KEY = 'subject'
+_EMAIL_SUBJECT_TEMPLATE = 'Nodejs Query Test Failure in {env}'
+_EMAIL_MESSAGE_KEY = 'message'
+_EMAIL_MESSAGE_TEMPLATE = 'There were diffs found when testing Nodejs Query results against goldens in {env}.<br><br><b>Goldens folder</b>: {goldens_path}<br><b>Nodejs Query Results</b>: {test_path}<br><b>Diff results</b>: {results_path}'
```

Comment:
Nice, could you add an example diff email screenshot?

Reply:
I have an example in the PR description, where did you want this example?

Reply:
Sorry, missed it.  Should we have the paths be like:

https://console.cloud.google.com/storage/browser/datcom-website-periodic-testing/dev

So they are clickable in the browser?

Reply:
done